### PR TITLE
upgrade to server-router 5.2

### PR DESCRIPTION
**Pull request by yoshuawuyts**
Upgrade to the latest server-router. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,8 +226,9 @@ Create a new instance of `merry`. Takes optional opts:
   write logs to
 - __opts.env:__ pass an object containing env var assertions
 
-### app.route(method, route, handler)
-Register a new handler for a route and HTTP method.
+### app.route(method|methods, route, handler)
+Register a new handler for a route and HTTP method. Method can be either a
+single HTTP method, or an array of HTTP methods.
```
Comment (lrlna) [inline README.md]: should we give an example as to how to do multiple methods, or is that too much

Comment (yoshuawuyts) [inline README.md]: I think it'd be too much

Comment (lrlna) [inline README.md]: 👌

**Comment by lrlna** [comment]
:package: `v5.3.0`

**Pull request merged**